ksqldb-rest-client/src/main/java/io/confluent/ksql/rest/client/KsqlRestClient.java

```diff
@@ -373,4 +359,18 @@ private static URI parseUri(final String serverAddress) {
           "The supplied serverAddress is invalid: " + serverAddress, e);
     }
   }
+
+  private Map<String, Object> setConsistencyVector(
+      final Map<String, ?> requestProperties
+  ) {
+    final Map<String, Object> requestPropertiesToSend = new HashMap<>(requestProperties);
+    if (ConsistencyOffsetVector.isConsistencyVectorEnabled(requestPropertiesToSend)) {
```

Comment:
This method is solely operating on the overrides map and the KsqlConfig, should we move it to Utils?

Reply:
It's only used in this class, I don't think this warrants moving it to a Utils class, which btw, I don't see exists in this package